packages/fxa-settings/src/pages/Signup/SignupConfirmedSync/index.tsx

```diff
@@ -0,0 +1,102 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { RouteComponentProps } from '@reach/router';
+import AppLayout from '../../../components/AppLayout';
+import Banner from '../../../components/Banner';
+import { useFtlMsgResolver } from '../../../models';
+import { FtlMsg } from 'fxa-react/lib/utils';
+import { hardNavigate } from 'fxa-react/lib/utils';
+import { firefox } from '../../../lib/channels/firefox';
+import { SignupConfirmedSyncProps } from './interfaces';
+import { SyncCloudsImage } from '../../../components/images';
+
+const SignupConfirmedSync = ({
+  integration,
+  // TODO check this was received from the webchannel message
+  paymentMethodsSynced = true,
+}: SignupConfirmedSyncProps & RouteComponentProps) => {
+  const ftlMsgResolver = useFtlMsgResolver();
+
+  // Currently, this page will only be shown to Sync Desktop users because once
+  // mobile users are signed in, the web view automatically closes; see FXA-10865.
+  // Once FxA operations in mobile are done in a normal tab (bz 1840491), this will
+  // be a non-issue, and support for the "manage sync" web channel message in mobile
+  // will have had time to roll out (FF 140+). When mobile _does_ show this page, we
+  // will want to look at the UX again, but for now we just won't show the pair flow link.
+  const showPairLink = integration.isDesktopSync();
+
+  return (
+    <AppLayout>
+      <Banner
+        type="success"
+        textAlignClassName="text-center"
+        content={{
+          localizedHeading: ftlMsgResolver.getMsg(
+            'signup-confirmed-sync-success-banner',
+            'Mozilla account confirmed'
+          ),
+        }}
+      />
+
+      <SyncCloudsImage className="mx-auto mt-4 max-h-44" />
+
+      <FtlMsg id="signup-confirmed-sync-header">
+        <h1 className="card-header">Sync is turned on</h1>
+      </FtlMsg>
+      {paymentMethodsSynced ? (
+        <FtlMsg id="signup-confirmed-sync-description-with-payment">
+          <p className="mt-2 mb-7">
+            Your passwords, payment methods, addresses, bookmarks, history, and
+            more can sync everywhere you use Firefox.
+          </p>
+        </FtlMsg>
+      ) : (
+        <FtlMsg id="signup-confirmed-sync-description">
+          <p className="mt-2 mb-7">
+            Your passwords, addresses, bookmarks, history, and more can sync
+            everywhere you use Firefox.
+          </p>
+        </FtlMsg>
+      )}
+
+      {showPairLink && (
```

Comment:
The linked issue says this should be "Start browsing", is that still correct?

Reply:
Sorry for not calling that out, no, it was changed because we can't open the new tab page without additional web channel message work; `window.open();` just opens `about:blank`, so Ross and Zach reworked this page a bit.